source/conf.py

```diff
@@ -447,7 +447,19 @@
     # Ignore Twitter links because for some reason valid links sometimes fail
     # with "400 Bad Request"
     r"^https://twitter.com/\w+$",
+    # Ignore support links because we get 403 Client Errors (bad User Agent?)
+    r"^https://support.native-instruments.com/",
+    r"^https://support.serato.com/",
```

Comment:
```suggestion
    r"^https://support.native-instruments.com/.*",
    r"^https://support.serato.com/.*",
```
otherwise it won't match for the subdirectories I think. We have to test this.

Reply:
it's a partial regex match (no $ at the end of line), I did test and it works.

Reply:
Great, I figured it doesn't work because pre-commit failed, but that was because the `linkcheck_ignore` list was overwritten.